doc/python/styling-plotly-express.md

```diff
@@ -0,0 +1,183 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.2'
+      jupytext_version: 1.3.1
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.6.8
+  plotly:
+    description: Plotly Express is a terse, consistent, high-level API for rapid data
+      exploration and figure generation.
+    display_as: file_settings
+    language: python
+    layout: base
+    name: Plotly Express
```

Comment:
Styling Plotly Express?

Reply:
ah yeah I forgot to actually edit the front matter here, it's a straight copy.